old-ui/app/css/index.css

```diff
@@ -654,7 +654,7 @@ input.large-input {
   overflow-x: hidden;
   /*overflow-y: auto;*/
   overflow-y: -moz-scrollbars-none;
-  max-height: 585px;
+  max-height: 568px;
```

Comment:
You shouldn't put a fixed value for this, instead of pixels we should be using em, rem or pt instead

Reply:
I dont remember changing this... i'll rollback to 585px as it was